dds2val/ddsproviderlib/idls/generate_py_dataclass.sh

```diff
@@ -2,17 +2,17 @@
 set -eu
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 cd "$SCRIPT_DIR"
-# moving files to tmp as the py files generated as on the same folder 
+# moving files to tmp as the py files generated as on the same folder
 # as the package name and would get deleted
 
 # git clone Vechicle signal specidification and vss-tools
 cd /tmp
 rm -rf /tmp/vehicle_signal_specification
 rm -rf /tmp/vss-tools
-git clone https://github.com/COVESA/vehicle_signal_specification.git -b v3.1.1
-git clone https://github.com/COVESA/vss-tools.git -b v3.1
+git clone https://github.com/COVESA/vehicle_signal_specification.git -b v4.0rc0
```

Comment:
NOTE: This needs to be changed to real tag later

We also have possibly found a bug in ddsidl exporter, build will likely fail